samples/compiler_plugins/simple_io_sample/test/print.mlir

```diff
@@ -0,0 +1,6 @@
+// RUN: iree-opt --iree-plugin=simple_io_sample --iree-print-plugin-info %s
+
+func.func @main() {
```

Comment:
Some CHECK? Perhaps a grep? Or relying just on whether passes can run or not?

Reply:
Currently the test doesn't even run because I need to get over some build and CI goo that is more easily iterated on when this lands.